packages/dashboard/src/CdcDashboardComponent.tsx

```diff
@@ -649,157 +650,219 @@ export default function CdcDashboard({ initialState, isEditor = false, isDebug =
       <>
         {isEditor && <Header />}
         <MultiTabs isEditor={isEditor && !isPreview} />
-        <div className={`cdc-dashboard-inner-container${isEditor ? ' is-editor' : ''}`}>
-          <Title title={title} isDashboard={true} classes={[`dashboard-title`, `${config.dashboard.theme ?? 'theme-blue'}`]} />
-          {/* Description */}
-          {description && <div className='subtext'>{parse(description)}</div>}
-
-          {/* Filters */}
-          {config.dashboard.sharedFilters && Object.values(config.visualizations || {}).filter(viz => viz.visualizationType === 'filter-dropdowns').length === 0 && (
-            <>
-              <Filters filters={state.config.dashboard.sharedFilters} apiFilterDropdowns={apiFilterDropdowns} handleOnChange={handleOnChange} />
-              <GoButton />
-            </>
-          )}
-
-          {/* Visualizations */}
-          {config.rows &&
-            config.rows
-              .filter(row => row.filter(col => col.widget).length !== 0)
-              .map((row, index) => {
-                const isToggleRow = row[0].toggle
-                return (
-                  <div className={`dashboard-row ${row.equalHeight ? 'equal-height' : ''} ${isToggleRow ? 'toggle' : ''}`} key={`row__${index}`}>
-                    {isToggleRow && <Toggle row={row} rowIndex={index} visualizations={config.visualizations} />}
-                    {row.map((col, colIndex) => {
-                      if (col.width) {
-                        if (!col.widget) return <div key={`row__${index}__col__${colIndex}`} className={`dashboard-col dashboard-col-${col.width}`}></div>
+        <Layout.Responsive isEditor={isEditor}>
+          <div className={`cdc-dashboard-inner-container${isEditor ? ' is-editor' : ''}`}>
```

Comment:
Did you make any changes here other than wrapping Layout.Responsive?

Reply:
No it should just be a container component wrapper.